.eslintrc.js

```diff
@@ -38,7 +38,8 @@ module.exports = {
     {
       files: ['*.js'],
       rules: {
-        '@typescript-eslint/no-var-requires': 'off'
+        '@typescript-eslint/no-var-requires': 'off',
+        '@typescript-eslint/no-unused-vars': 'off'
```

Comment:
Uh I don't want to get you on your nerves with that, trust me ^^'

But you should swap these two lines, because in the other PRs we also have them this way and `u` is one letter before `v` in alphabetical order
So we wont have merge conflicts for this single line